tests/subsys/bluetooth/fast_pair/storage/factory_reset/src/main.c

```diff
@@ -320,6 +348,13 @@ static void fp_storage_validate_empty(void)
 	zassert_ok(err, "Unexpected error in Account Key get");
 	zassert_equal(read_cnt, 0, "Invalid Account Key count");
 
+	err = fp_storage_clock_boot_checkpoint_get(&clock_checkpoint);
```

Comment:
Currently we do not test getting boot checkpoint well. We should consider adding a small test to validate if the boot checkpoint is updated only once during the first initialization.

Reply:
Created a ticket for this additional work item:

https://nordicsemi.atlassian.net/browse/NCSDK-27439

Please do not resolve this issue as your comment is linked in the ticket description

Reply:
Well, I decided to extend the implementation a bit to also cover this part as it was a small change after all.